MOBILE_CLIENT/COMMON/_CRITICAL/MALICIOUS_PACKAGE_COMOUTSYSTEMS_FILEVIEWER/description.md

```diff
@@ -0,0 +1,7 @@
+The package `com.outsystems.plugins.fileviewer` version `1.0.6` has been identified as malicious. This specific version, tracked under the identifier `MAL-2022-2047`, poses a severe security risk.
```

Comment:
Nope should be generic, not about this package, about malicious packages in general.